Lama(Databases-week1)

LamaGharib opened the pull request.

unmeshvrije commented on Week1/meetup_query.js:
```diff
@@ -0,0 +1,38 @@
+        room_name VARCHAR(50),
+        floor_number INT)`;
+const create_table_Meeting = `CREATE TABLE meetup.Meeting(
+            meeting_no INT PRIMARY KEY AUTO_INCREMENT,
```
Too much space here. try to be consistent with previous queries and spacing

unmeshvrije commented on Week1/meetup_query.js:
```diff
@@ -0,0 +1,38 @@
+('Jenny Doe','Sam Smith'),
+('July Doe','Sam Smith')`;
+const populate_room = `INSERT INTO meetup.Room(room_name,floor_number)
+VALUES ('A',3),('B',2),('C',6),('D',3),('E',5)`;
```
Too little space here. Again the same comment about consistency of spacing

unmeshvrije commented on Week1/world_query.js:
```diff
@@ -0,0 +1,22 @@
+const tenMostPopulatedCities = `SELECT name FROM city ORDER BY population DESC LIMIT 10`;
+const populationOfTheWorld = `SELECT SUM (population) AS 'TOTAL_POPULATION' FROM country`;
+module.exports = {
+  countriesPopulationGreaterThan8000000,
```
Good names with camel case convention. You could make them shorter e.g. `countriesPopMoreThan8M`

unmeshvrije reviewed (changes requested):
Good job! Left some comments , please go through them :)

LamaGharib commented:
Hey, @unmeshvrije thanks for the feedback and sorry for the super late reply.
I removed the spacing at CREATE TABLE and added some spacing at VALUES, I'm not sure if that's what u meant, coz for me what I had was more readable (i mean the create table part )
